x-pack/plugins/security_solution/public/management/pages/event_filters/constants.ts

```diff
@@ -15,7 +19,7 @@ import {
 } from '@kbn/securitysolution-list-constants';
 
 export const EVENT_FILTER_LIST_TYPE: ExceptionListType = ExceptionListTypeEnum.ENDPOINT_EVENTS;
-export const EVENT_FILTER_LIST = {
+export const EVENT_FILTER_LIST: CreateExceptionListSchema = {
```

Comment:
I think in my PR I also renamed the variable to `*.DEFINITION` so depending on which of us merges first, lets make sure we `merge upstream` on the PR before merging